src/types/DateSBVR.ts

```diff
@@ -0,0 +1,19 @@
+import * as TypeUtils from '../TypeUtils'
+type InternalDate = Date | number
+
+export class DateSBVR implements SBVRType<Date, Date> {
+	types = {
+		postgres: 'DATE',
+		mysql: 'DATE',
+		websql: 'INTEGER',
+		odata: {
+			name: 'Edm.DateTime'
+		},
+	}
+
+	fetchProcessing = (data:InternalDate, callback:Callback<Date>) => {
```

Comment:
`data` can also be null and should be handled (otherwise it will return a date of 1970)